web_programming/instagram_crawler.py

```diff
@@ -0,0 +1,176 @@
+#!/usr/bin/python3
+# -*- coding: utf-8 -*-
+import requests
+from bs4 import BeautifulSoup
+import json
+
+headers = \
+    {
+        'User-Agent': 'Mozilla/5.0 (Macintosh; Intel Mac OS X 10_12_5) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/58.0.3029.110 Safari/537.36'}
+
+# Usage
+"""
+>>> user = Instagram("github")
+>>> user.is_verified()
+True
+>>> user.get_biography()
+Built for developers.
+
+"""
+
+
+class Instagram(object):
```

Comment:
```suggestion
class InstagramUser:
```
`(object)` is longer required in modern Python.